### DISCUSSION TITLE
v5.7.0

### DISCUSSION THREAD

**Discussion by niden** [phalcon/cphalcon#16624] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/phalcon/cphalcon/releases/tag/v5.8.0'>v5.8.0</a>.</em>

**Comment by s-ohnishi** [comment]
Isn't it v5.8.0?

**Reply by niden** [reply]
Yes thank you. The label is wrong.

**Comment by okamaxomorra** [comment]
I have problem with Devtools. This is the info:

_Phalcon DevTools (4.2.0)

Environment:
  OS: Windows NT LAPTOP-HRV5F05B 10.0 build 22631 (Windows 11) AMD64
  PHP Version: 8.2.12
  PHP SAPI: cli
  PHP Bin: D:\php82\php\php.exe
  PHP Extension Dir: C:\php\ext
  PHP Bin Dir: C:\php
  Loaded PHP config: D:\php82\php\php.ini
Versions:
  Phalcon DevTools Version: 4.2.0
  Phalcon Version: 5.8.0
  AdminLTE Version: 3.0.1_
  
 I tried to create new project error found:
 
```
 # phalcon project --name=myapi --directory=D:\php82\htdocs --type=micro

Phalcon DevTools (4.2.0)

PHP Fatal error:  Uncaught Error: Class "Phalcon\Config" not found in D:\php82\phalcon5\vendor\phalcon\devtools\src\Builder\Component\AbstractComponent.php:48
Stack trace:
#0 D:\php82\phalcon5\vendor\phalcon\devtools\src\Commands\Builtin\Project.php(66): Phalcon\DevTools\Builder\Component\AbstractComponent->__construct(Array)
#1 D:\php82\phalcon5\vendor\phalcon\devtools\src\Script.php(109): Phalcon\DevTools\Commands\Builtin\Project->run(Array)
#2 D:\php82\phalcon5\vendor\phalcon\devtools\src\Script.php(148): Phalcon\DevTools\Script->dispatch(Object(Phalcon\DevTools\Commands\Builtin\Project))
#3 D:\php82\phalcon5\vendor\phalcon\devtools\phalcon(65): Phalcon\DevTools\Script->run()
#4 {main}
  thrown in D:\php82\phalcon5\vendor\phalcon\devtools\src\Builder\Component\AbstractComponent.php on line 48

Fatal error: Uncaught Error: Class "Phalcon\Config" not found in D:\php82\phalcon5\vendor\phalcon\devtools\src\Builder\Component\AbstractComponent.php:48
Stack trace:
#0 D:\php82\phalcon5\vendor\phalcon\devtools\src\Commands\Builtin\Project.php(66): Phalcon\DevTools\Builder\Component\AbstractComponent->__construct(Array)
#1 D:\php82\phalcon5\vendor\phalcon\devtools\src\Script.php(109): Phalcon\DevTools\Commands\Builtin\Project->run(Array)
#2 D:\php82\phalcon5\vendor\phalcon\devtools\src\Script.php(148): Phalcon\DevTools\Script->dispatch(Object(Phalcon\DevTools\Commands\Builtin\Project))
#3 D:\php82\phalcon5\vendor\phalcon\devtools\phalcon(65): Phalcon\DevTools\Script->run()
#4 {main}
  thrown in D:\php82\phalcon5\vendor\phalcon\devtools\src\Builder\Component\AbstractComponent.php on line 48
```

**Comment by HiralRajgor** [comment]
Hi, @niden 
it's high priority, i m using alpine linux, and in docker i have mentioned this package

RUN apk add php82-pecl-phalcon --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing

it was working fine before few hours, but suddenly it says package not found. we tried alternatives, but not working
Please help me asap, my production deployments are stopped for all the projects.

ERRORS in my docker build
--
ERROR: unable to select packages:
php82-pecl-phalcon (no such package):
required by: world[php82-pecl-phalcon]
ERROR: process "/bin/sh -c apk add php82-pecl-phalcon --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing" did not complete successfully: exit code: 1
------

**Reply by niden** [reply]
Sorry we are not responsible or have anything to do with alpine packages. We just publish to PECL and then the extension propagates to distributions.
